doc/tutorial/alignment_file/alignment_file_read_cigar.cpp

```diff
@@ -34,9 +34,9 @@ int main()
 {
     std::filesystem::path tmp_dir = std::filesystem::temp_directory_path(); // get the temp directory
 
-    seqan3::alignment_file_input fin{tmp_dir/"my.sam", seqan3::fields<seqan3::field::cigar>{}};
+    seqan3::alignment_file_input fin{tmp_dir/"my.sam"}; // default fields
 
-    for (auto & [cigar_vector] : fin)
-        seqan3::debug_stream << cigar_vector << '\n';
+    for (auto & rec : fin)
```

Comment:
I would not change this snippet. But rather add a new one showing that it is also present in the default fields. Is there no snippet for this as well?

Reply:
This is a snippet in the alignment I/O tutorial explaining how to read the cigar string. 
That's why I used the default fields now to show that is included by default. 
I don't know where I should add another snippet. I can add a comment `you could also select the field::cigar specifically` but selecting custom fields is covered in an extra section.

Reply:
Mhmm, I thought there is at least one with the default fields. But that is fine with me here as well.